pkgs/development/python-modules/scikit-survival/default.nix

```
@@ -0,0 +1,71 @@
+{ lib
+, buildPythonPackage
+, cython
+, ecos
+, fetchPypi
+, joblib
+, numexpr
+, numpy
+, osqp
+, pandas
+, pytestCheckHook
+, scikitlearn
+, scipy
```

Most packages tend to sort these by group/type of package, so it should be something like
```suggestion
, fetchPypi
, cython
, ecos
, joblib
, numexpr
, numpy
, osqp
, pandas
, scikitlearn
, scipy
  # Test inputs
, pytestCheckHook
```